src/styles/index.css

```diff
@@ -18,6 +18,10 @@ html {
 
 svg.recharts-surface {
   overflow: visible;
+  position: absolute;
+  top: 0;
+  left: 24px;
```

Comment:
This affects all charts. Do charts other than staking trend have the same issue?